src/librustc_codegen_llvm/lib.rs

```diff
@@ -79,6 +79,7 @@ use rustc::ty::{self, TyCtxt};
 use rustc::util::nodemap::{FxHashSet, FxHashMap};
 use rustc_mir::monomorphize;
 use rustc_codegen_utils::codegen_backend::CodegenBackend;
+use rustc_codegen_utils::time_graph;
```

Comment:
Moved to `rustc_codegen_utils`, because it is useful for other future backends too.